### PR TITLE
[Wait for v1.1 release] Add a Query example for reverse

### PR DESCRIPTION
**Pull request by MichelDiz**
It also modifies some points in the schema to fit the query. And I added a note about version 1.1.

### CODE REVIEW

**Comment by danielmai** [comment]
We can still have `_reverse_` queries for pre-v1.1, right? The only distinction I see is the `[uid]` syntax in the schema.

**Comment by MichelDiz** [comment]
Yep, But I think there's no hurry on this.

**Comment by paulftw** [comment]
@MichelDiz @prashant-shahi same here - we don't need to wait for 1.1, this PR can be merged now.

**Comment by MichelDiz** [comment]
I'll work in some conflicts with this.

**Comment by prashant-shahi** [comment]
I'll send a new PR from my branch to master.

File: content/schema/10.md [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+next = "/schema/11"
 prev = "/schema/9"
-title = "Congratulations"
+title = "Expand Predicate"
 weight = 10
 +++
 
-This is the end of the schema module.
+`expand(...predicates...)` is used to query for all given predicates,
+rather than listing them in the query.  Calling
```
Comment (mangalaman93) [inline content/schema/10.md]: `Calling` -> `Querying`

File: content/schema/10.md [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
+```
+expand(_all_)
+```
+queries for all edges out of every node matched at that level in the
```
Comment (mangalaman93) [inline content/schema/10.md]: `queries` -> `returns`

**Review by mangalaman93 — APPROVED** [review]

File: content/schema/6.md [hunk 3]
```diff
@@ -1,31 +1,11 @@
...
-```
-
-Because the `uid` picked by Dgraph is unique, we can't help you this
-time.  Use the `uid`'s picked by your instance of Dgraph to  write a mutation that links the company and
-friendship data.  Hint: previous queries will tell you the `uid`.
-
-The process you've just done here would normally done
-programmatically - query the data to get the `uid`'s, formulate the
-mutations, then batch  the updates.
+In this query we want to know who works for "CompanyABC" without having to add extra edges. So we use a reverse edge for the specific case. And then we use an alias "work_here" to differentiate the query result.
```
Comment (mangalaman93) [inline content/schema/6.md:11]: 100 chars

File: content/schema/7.md [hunk 4]
```diff
@@ -1,74 +1,31 @@
...
+We've added a new schema and loaded some company data, but what about
+integrating our previous friends dataset with this company one.
 
-* `<uid> <edge> <uid>/"value" .`  Delete a single triple
-* `<uid> <edge> * .`  Delete all triples for a given edge
-*  `<uid> * * .` Delete all triples for a given node
+Trying to use the blank nodes from previous mutations won't work.  The blank nodes aren't persisted in the store, so when referring to nodes created in a previous mutation, it's the UID that's needed.  So instead of
 
-{{% notice "note" %}}
-  In order to delete with `S * *` pattern. You need to have a Type related to the node itself. See: [Type System](https://docs.dgraph.io/master/query-language/#type-system)
```
Comment (mangalaman93) [inline content/schema/7.md:17]: We should have a similar note like this in the tutorial.

File: content/schema/6.md (hunk 3 above)
Comment (MichelDiz) [reply content/schema/6.md:11]: I did not understand the comment

File: content/schema/7.md (hunk 4 above)
Comment (MichelDiz) [reply content/schema/7.md:17]: I didn't understand this one either. I think this comment seems to be to the schema/8.md

**Pull request merged**